src/main/flight/gps_rescue.c

```diff
@@ -458,9 +458,11 @@ static void performSanityChecks()
         }
     }
 
-    // Check if crash recovery mode is active
+    // If crash recovery is triggered during a rescue, immediately disarm, ignoring the crash flip recovery settings.
     if (crashRecoveryModeActive()) {
-        rescueState.failure = RESCUE_CRASH_FLIP_DETECTED;
```

Comment:
`RESCUE_CRASH_FLIP_DETECTED` is not used anywhere. Please also remove definition from line 79.